docs/lake/design.md

```diff
@@ -147,33 +151,42 @@ decomposed into two steps: an `add` operation and a `commit` operation.
 These steps can be explicitly run in stages, e.g.,
 ```
 zed lake add -p logs sample.json
-(commit <tag> printed to stdout)
+(commit <tag> etc printed to stdout)
 zed lake commit -p logs <tag>
 ```
-A commit `<tag>` refers to one or more data objects stored in the
-data pool.  Genreally speaking, a list of commit tags is simply a shortcut
-for the set of object tags that comprise the commits.  Both commit and object
-tags are named using the same globally unique allocation of
-[KSUIDs](https://github.com/segmentio/ksuid).
+A commit `<tag>` refers to one or more data segments stored in the
+data pool.  In general, a commit tag is simply a shortcut
+for the set of object tags that comprise the commit and otherwise
+has no meaningful semantics to the Zed execution engine.
+Both commit and data tags are named using the same globally unique
+allocation of [KSUIDs](https://github.com/segmentio/ksuid).
```

Comment:
We might want to follow Git's lead here and use "name" in place of "tag" for these terms, for two reasons. First, a Git user's notion of tags is quite different from this one, and that might cause confusion. And second, we might want to add Git-style tags some day.

Reply:
Good idea.  I'll create an issue since it's also used throughout the command-tool help.

Issue #2590